hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/handler/CloseMetaHandler.java

```diff
@@ -31,10 +32,8 @@
 public class CloseMetaHandler extends CloseRegionHandler {
 
   // Called when regionserver determines its to go down; not master orchestrated
-  public CloseMetaHandler(final Server server,
```

Comment:
I actually prefer these long method argument lists being a single parameter per line, but our style guides don't agree.

Reply:
Right. Mine agrees w/ general code base. Yours does not (smile).

Reply:
_nod_